modules/src/xibo-elements-render.js

```diff
@@ -136,7 +136,7 @@ jQuery.fn.extend({
         ((options.effect === 'marqueeLeft') ? 'left' : 'right');
 
       // Make sure the speed is something sensible
-      options.speed = (options.speed === 0) ? 1 : options.speed;
+      options.speed = (options.speed === 0) ? 1 : options.speed / 25 * 1000;
```

Comment:
What is the reason for dividing by 25, and can be add a comment explaining that?

Reply:
This calculation gives us a result of `80` as a value for `options.speed`. This value is the speed we are already using from `xibo-text-render.js` which is I believe based on the **85 milliseconds** default scrollamount as pixels per second.

Reply:
Understood! I think a comment explaining this for "future Ruben" would be awesome, but you can merge this for testing in the meantime.

Reply:
Yep, got it @dasgarner. I'll take this into account. Thanks!